MangoAPI.Presentation/Startup.cs

```diff
@@ -80,6 +81,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
         app.Map(RoutingConstants.Login, builder => builder.UseSpa(spa => spa.Options.SourcePath = "/wwwroot"));
 
         app.MigrateDatabase();
+        Task.FromResult(app.InitializeAzureBlobAsync());
```

Comment:
I don't like this statement, if you cannot call async API normally, then implement your migrator in synchronous way and call it properly